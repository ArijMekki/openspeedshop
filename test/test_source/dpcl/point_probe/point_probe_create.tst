#! /bin/tcsh
./mutator 301 hope2.americas.sgi.com $cwd/mutatee 301
