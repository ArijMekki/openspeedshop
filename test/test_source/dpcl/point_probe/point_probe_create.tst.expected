Doing 30 calls to sleep
Found symtab
Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Hello world from myprobe module

Counted 30 calls to sleep()!
SUCCESS
